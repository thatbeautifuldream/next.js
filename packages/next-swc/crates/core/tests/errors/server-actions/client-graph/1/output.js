/* __next_internal_client_entry_do_not_use__ default auto */ /* __next_internal_action_entry_do_not_use__ $$ACTION_0 */ import __create_action_proxy__ from "private-next-rsc-action-proxy";
export default function App() {
    async function fn(...args) {
        return $$ACTION_0.apply(null, (fn.$$bound || []).concat(args));
    }
    __create_action_proxy__("6d53ce510b2e36499b8f56038817b9bad86cabb4", null, fn, $$ACTION_0);
    return <div>App</div>;
}
export async function $$ACTION_0() {}
