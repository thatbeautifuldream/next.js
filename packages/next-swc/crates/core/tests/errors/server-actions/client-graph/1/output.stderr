
  x "use server" functions are not allowed in client components. You can import them from a "use server" file instead.
   ,-[input.js:3:1]
 3 |     export default function App() {
 4 | ,->   async function fn() {
 5 | |       'use server'
 6 | `->   }
 7 |       return <div>App</div>
   `----
